### PR TITLE
Project standardization

### PR DESCRIPTION
**Pull request by bandophahita**
https://github.com/ScreenPyHQ/screenpy/issues/127
Attempting to make various portions of each screenpy project uniform.

### CODE REVIEW

File: screenpy_requests/__version__.py [hunk 1]
```diff
@@ -1,6 +1,6 @@
 """
-█▀ █▀▀ █▀█ █▀▀ █▀▀ █▄░█ █▀█ █▄█   █▀█ █▀▀ █▀█ █░█ █▀▀ █▀ ▀█▀ █▀
-▄█ █▄▄ █▀▄ ██▄ ██▄ █░▀█ █▀▀ ░█░   █▀▄ ██▄ ▀▀█ █▄█ ██▄ ▄█ ░█░ ▄█
+            █▀ █▀▀ █▀█ █▀▀ █▀▀ █▄░█ █▀█ █▄█   █▀█ █▀▀ █▀█ █░█ █▀▀ █▀ ▀█▀ █▀
+            ▄█ █▄▄ █▀▄ ██▄ ██▄ █░▀█ █▀▀ ░█░   █▀▄ ██▄ ▀▀█ █▄█ ██▄ ▄█ ░█░ ▄█
```
Comment (perrygoy) [inline screenpy_requests/__version__.py:3]: 👏 👏 👏

File: pyproject.toml [hunk 2]
```diff
@@ -15,111 +34,20 @@ extend-exclude = '''
...
-
 
 [tool.ruff]
 target-version = "py38"  # minimum supported version
 line-length = 88  # same as Black.
-
 extend-exclude = [
-    "screenpy_requests/__init__.py",
-    "screenpy_requests/__version__.py",
+    "__init__.py",
```
Comment (perrygoy) [inline pyproject.toml]: Same thing here about the `__init__.py` i mentioned in ScreenPy: Playwright. I don't think we want to ignore *all* of them, and we can probably just remove this exclusion.

**Review by perrygoy — COMMENTED** [review]
Just a comment about the `__init__.py` exclusion!

File: pyproject.toml (hunk 2 above)
Comment (bandophahita) [reply pyproject.toml]: Updated.

**Review by perrygoy — APPROVED** [review]
Phew, thanks @bandophahita! It's looking great.

**Pull request merged**